build.sbt

```diff
@@ -871,24 +871,11 @@ lazy val bench = http4sProject("bench")
   )
   .dependsOn(core, circe, emberCore)
 
-lazy val docs = http4sProject("docs")
-  .enablePlugins(
-    GhpagesPlugin,
-    NoPublishPlugin,
-    ScalaUnidocPlugin,
-    MdocPlugin,
-    LaikaPlugin,
-  )
-  .settings(docsProjectSettings)
+lazy val unidocs = http4sProject("unidocs")
+  .enablePlugins(TypelevelUnidocPlugin)
```

Comment:
I forgot to mention this: this is a new artifact `http4s-docs` that publishes the aggregated unidocs so they can be viewed at javadoc.io.

Reply:
I have been wondering what will become of the unidocs in the Great Schism.  Since the point is to be on different release cycles, I don't think there is much we can do.  So the value of this will drop, but I think it still provides incremental value.

Reply:
We could set up an auto-repo that gets steward PRs for latest http4s modules, extracts their API docs, and publishes them to `api.http4s.org` or something. Of course, it wouldn't be able to have any reasonable versioning since it aggregates across independently-versioned modules. But, it could still be useful.

Reply:
Until we publish a new release of v0.22, the API url on the site will be broken because the unidoc `http4s-docs` artifact does not exist yet. We could temporarily hard-code it to something else, but I'd be happy to see a release train start right after we merge this.

Reply:
I think a new release of 0.22 is coming in the next day or so.  There's the cookie thing and the ember client thing for 0.23.